client/src/app/management/components/member-list/member-list.component.html

```diff
@@ -56,8 +56,30 @@ <h2>{{ 'Members' | translate }}</h2>
             </os-icon-container>
         </div>
     </div>
+    <!-- Menu column -->
+    <div *pblNgridCellDef="'utils'; row as user" class="cell-slot fill">
+        <div>
+            <button mat-icon-button [matMenuTriggerFor]="singleMenu" [matMenuTriggerData]="{ member: user }">
+                <mat-icon>more_vert</mat-icon>
+            </button>
+        </div>
+    </div>
+    <ng-container *osOmlPerms="OML.can_manage_users"> </ng-container>
 </os-basic-list-view-table>
 
+<mat-menu #singleMenu="matMenu">
+    <ng-template matMenuContent let-member="member">
+        <button mat-menu-item (click)="navigateToMember(member)">
+            <mat-icon>edit</mat-icon>
+            <span>{{ 'Edit' | translate }}</span>
+        </button>
```

Comment:
navigations are prefarably `a` tags whenever possible, so you can open them in a new tab. Simply extract the url the the members edit page